x/emissions/keeper/keeper.go

```diff
@@ -57,6 +57,14 @@ type Keeper struct {
 	// for an address, what are all the topics that it's registered for?
 	addressTopics collections.Map[sdk.AccAddress, []uint64]
 
+	/// SCORES
+	// map of (topic, block_number, worker) -> score
+	workerInferenceScores collections.Map[collections.Pair[TOPIC_ID, BLOCK_NUMBER], types.Scores]
+	// map of (topic, block_number, worker) -> score
+	workerForecastScores collections.Map[collections.Pair[TOPIC_ID, BLOCK_NUMBER], types.Scores]
+	// map of (topic, block_number, reputer) -> score
+	reputerScores collections.Map[collections.Pair[TOPIC_ID, BLOCK_NUMBER], types.Scores]
+
```

Comment:
maybe we can save some letters and just call this inferenceScores, forecastScores, reputerScores?  

or not, just seems a bit verbose